### ISSUE TITLE
Update to Makie 0.20

### ISSUE THREAD

**Issue by ffevotte** [maltezfaria/DataFlowTasks.jl#88]
It looks like the profile plot must be updated for Makie 0.20, at least to avoid using the deprecated `resolution` keyword. There are possibly other tweaks to make...

**Comment by maltezfaria** [comment]
I tried it locally and it worked... maybe I was on the wrong version of `Makie`

I am going to revert the last commit on `main`, and we can later open a *PR* to fix this.

**Comment by ffevotte** [comment]
It's OK for now: it's merely a warning and we can always fix this later. I think we just don't want to release a patch version without fixing this first.

**Comment by maltezfaria** [comment]
Ok. The `CI` did work with the latest `Makie`:

https://github.com/maltezfaria/DataFlowTasks.jl/actions/runs/6978168705/job/18989110560

**Comment by ffevotte** [comment]
Yes. The deprecation warning can be seen e.g. here, in cell [10]: https://nbviewer.org/github/maltezfaria/DataFlowTasks.jl/blob/gh-pages/dev/examples/cholesky/cholesky.ipynb

But (1) it works, and (2) I think the warning does not even show up in the documentation pages. So we're fine for now: no need to revert the commit IMO, but no need to rush to a fix either.
